ethtx_ce/frontend/deps.py

```diff
@@ -30,12 +32,19 @@
 @auth.verify_password
 def verify_password(username: str, password: str) -> bool:
     """Verify user, return bool."""
-    return (
-        username == Config.ETHTX_ADMIN_USERNAME
-        and password == Config.ETHTX_ADMIN_PASSWORD
+    return username == Config.ETHTX_ADMIN_USERNAME and compare_digest(
+        password, Config.ETHTX_ADMIN_PASSWORD
     )
 
 
+def get_eth_price() -> Optional[float]:
+    response = requests.get("https://api.coinbase.com/v2/prices/ETH-USD/buy")
+
+    if response.status_code == 200:
+        eth_price = float(json.loads(response.content)["data"]["amount"])
+        return eth_price
+
```

Comment:
Guess we can return 0 at least if coinbase is not responding

Reply:
Done